### ISSUE TITLE
The SSH configuration in Settings cannot save the password and you need to enter the password manually each time

### ISSUE THREAD

**Issue by Austzhu** [felixse/FluentTerminal#833]
As stated in the title

**Comment by skipmeister123** [comment]
As a security professional, I can emphatically say: do not trust any SSH 
client which saves your password, as it would have to be saved in plaintext.

Your best (and only real) solution is to use a key.

I would strongly object to any changes in FT that facilitate saving the 
password.

**Comment by shaqaruden** [comment]
maybe integration with password managers?

**Comment by shaqaruden** [comment]
Wanted to comment on your reasoning against this request. A client does not HAVE to store your password in plain text. They could hash it for storage and resolve it when needing to use it.

**Comment by skipmeister123** [comment]
That might be a more acceptable solution, but then the question becomes: which password manager(s)?  And are there any password managers with an API?  I hope not, because that would inherently defeat the security (allowing unrestricted background access via IPC).

**Comment by skipmeister123** [comment]
That's true, but if the method of hashing is reversible (which it would need to be for this), then what's the point in hashing it in the first place?

**Comment by atomrq** [comment]
Hi, I use it only in local net for computation servers. I hope "password" function can be avilable, since it is covinient for such situation. And I think the responsibility to insure password security should be set to users not developers. Thank you!

**Comment by hanskokx** [comment]
I would point out that the code is open source. Anyone so inclined could 
make these changes for themselves.  However, for previously stated reasons, 
this is unlikely to be supported.  Just because you can doesn't mean you 
should.  You can give a toddler a loaded gun, but it doesn't mean you 
should. I will always advocate for users who might not know any better, or 
who don't know as much about security as someone with a degree in it, such 
as me.

**Comment by faisal6621** [comment]
Maybe a plugin in [KeePass](https://keepass.info/help/v2_dev/plg_index.html) to open directly the FT could help.
